saleor/attribute/tasks.py

```diff
@@ -29,3 +30,12 @@ def update_associated_products_search_vector(attribute_value_pk: int):
         Q(Exists(instance.productassignments.filter(product_id=OuterRef("id"))))
         | Q(Exists(variants.filter(product_id=OuterRef("id"))))
     ).update(search_index_dirty=True)
+
+
+@app.task()
+def setup_max_sort_order():
+    for attribute in Attribute.objects.filter(max_sort_order=None).iterator():
```

Comment:
I think it needs to use batching as well. If you had say a million attributes, it would work well with the memory when we have an iterator, but the task could still take a long time. Better would be to have smaller tasks that queue themselves when needed and operate on batches.

Another question is if we could do that in a single `update` without looping?